views/home/index.ejs

```diff
@@ -86,8 +85,47 @@
               >
                 Generate
               </button>
+              <!-- Button to trigger the help modal -->
+              <button
+                id="modal-btn"
+                class="btn btn-outline-secondary btn-m"
+                type="button"
+                aria-label="information"
+              >
+                <i id="info-button" class="fa-solid fa-circle-question"></i>
+              </button>
+              <!-- Help modal -->
+              <div class="modal" aria-modal="true">
+                <div class="modal-content">
+                  <span class="close-btn">&times;</span>

```

Comment:
This doesn't change the cursor to a clicker, but it might not be a big deal if we aren't letting people test this on the desktop

Reply:
fixed it to pointer!